client/my-sites/backup/backup-contents-page/file-browser/index.tsx

```diff
@@ -75,18 +66,8 @@ function FileBrowser( {
 
 	return (
 		<div>
-			{ ( fileBrowserConfig?.showHeader ?? true ) && (
-				<FileBrowserHeader
-					rewindId={ rewindId }
-					siteId={ siteId }
-					siteSlug={ siteSlug }
-					hasCredentials={ hasCredentials }
-					isRestoreEnabled={ isRestoreEnabled }
-					onTrackEvent={ onTrackEvent }
-					onRequestGranularDownload={ onRequestGranularDownload }
-					onRequestGranularRestore={ onRequestGranularRestore }
-				/>
-			) }
+			{ ( fileBrowserConfig?.showHeader ?? true ) && <FileBrowserHeader /> }
+			{ /* @TODO: remove this block once the new Staging Sync Modal is live */ }
```

Comment:
cc/ @Automattic/yolo for visibility. I think we can remove the `showBackupTime` parameter from the fileBrowserConfig and the `displayBackupDate` once the new Staging Sync Modal is live, since `showBackupTime` is now set to false:

https://github.com/Automattic/wp-calypso/blob/be01623b977330919a28ca2a291fb27cd3aba10d/client/dashboard/sites/staging-site-sync-modal/index.tsx#L45

And the displayBackupDate is no longer passed to the FileBrowser:

https://github.com/Automattic/wp-calypso/blob/be01623b977330919a28ca2a291fb27cd3aba10d/client/dashboard/sites/staging-site-sync-modal/index.tsx#L444-L449

I'm not removing them yet since it is still in use for the V1.

That sounds good to you?